launch/launch/frontend/expose.py

```diff
@@ -40,13 +40,13 @@ def instantiate_action(entity: 'Entity', parser: 'Parser') -> Action:
 
 def instantiate_substitution(
     type_name: Text,
-    args: Iterable[SomeSubstitutionsType]
+    args: Iterable[SomeSubstitutionsType] = []
```

Comment:
This is a usually a bad idea, though in this case I think it's technically safe, it would be better to avoid it, see:

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Thanks! I didn't know that.